api/wasm/cpp/proxy_wasm_api.h

```diff
@@ -513,6 +513,78 @@ inline bool getMessageValue(std::initializer_list<StringView> parts, T* value_pt
   return value_ptr->ParseFromArray(buf.value()->data(), buf.value()->size());
 }
 
+// Create an expression using a foreign function call.
+inline WasmResult exprCreate(StringView expr, uint32_t* token) {
+  std::string function = "expr_create";
```

Comment:
Naming: Is this just expr_evaluate or cel_expr_evaluate?

Reply:
Fine with either. compressor is not using zip in its name, so I thought I'd do the same.